src/epics/router.ts

```diff
@@ -130,12 +133,47 @@ const changeView: Epic = (action$, store) => action$.ofType(types.selectView)
 
 const drillDown: Epic = (action$, store) => action$.ofType(types.drillDown)
 .switchMap(action => {
-    // const state = store.getState() as Store
+    const state = store.getState() as Store
     const url = action.payload.url
     switch (action.payload.drillDownType) {
-        case 'inner':
+        case DrillDownType.external:
+            window.location.href = url
+            return Observable.empty()
+        case DrillDownType.externalNew:
+            if (/^[a-z0-9]+:\/\//i.test(url)) {
```

Comment:
Will it work for urls without protocol specified, e.g. "www.github.com"?

Reply:
"window.open" always opens relative url if it came without a protocol